### ISSUE TITLE
Restore "Copy image" in context menu

### ISSUE THREAD

**Issue by kjk** [sumatrapdfreader/sumatrapdf#2616]
### Discussed in https://github.com/sumatrapdfreader/sumatrapdf/discussions/2615

<div type='discussions-op-text'>

<sup>Originally posted by **nicamoq** May 28, 2022</sup>
Is it possible to bring back the "Copy image" option below "Copy selection" in the context menu? There doesn't seem to be a way to copy a whole page anymore.

</div>

**Comment by nicamoq** [comment]
> Actually, I can't reproduce. Can you provide a file that works differently in 3.4 vs. 3.3 and exact reproduction steps in #2616

As additional info, I'm using 3.4.2 portable x64 updated from 3.3.3 by simply replacing the 3.3.3 .exe, thus using the same cache and config file. But I also tried it in a fresh portable .exe with a rebuilt config.

I found that _all_  .zip and .cbz files don't have "Copy image" in the context menu. Opening an image file directly also doesn't have it. However, all my .epub and .pdf files still do.


Here's a screenshot from a fresh SumatraPDF-3.4.2-64.exe when right-clicking an opened .png file:
![image](https://user-images.githubusercontent.com/27849412/170835300-fc92d0a5-60f3-461b-b09a-b255f8def1d8.png)
